modules/claims_api/app/swagger/claims_api/v1/form_526_controller_swagger.rb

```diff
@@ -338,13 +346,15 @@ class Form526ControllerSwagger
             key :type, :string
           end
 
-          parameter do
+          request_body do
```

Comment:
I like this but make sure it renders properly on the portal's version of Swagger UI.

Reply:
looks like it does ok